frontend/composables/useCustomFetch.ts

```diff
@@ -38,6 +38,7 @@ export function useCustomFetch () {
     const { public: { apiClient, legacyApiClient, apiKey, domain, logIp }, private: pConfig } = runtimeConfig
     const path = map.mock ? `${pathName}.json` : map.getPath?.(pathValues) || map.path
     let baseURL = map.mock ? '../mock' : map.legacy ? legacyApiClient : apiClient
+    const ssrSecret = pConfig?.ssrSecret
```

Comment:
nitpick: all `runtimeConfigs` that are not part of `public` are `private`, so this could be simplified
```suggestion
    const { public: { apiClient, legacyApiClient, apiKey, domain, logIp }, ssrSecret } = runtimeConfig
    const path = map.mock ? `${pathName}.json` : map.getPath?.(pathValues) || map.path
    let baseURL = map.mock ? '../mock' : map.legacy ? legacyApiClient : apiClient
```